spec/lib/arclight/normalized_title_spec.rb

```diff
@@ -119,18 +119,6 @@
     let(:title) { nil }
     let(:date) { nil }
 
-    it 'uses the id' do
-      expect(normalized_title).to eq id
-    end
-  end
-
-  context 'no information' do
-    let(:title) { nil }
-    let(:date) { nil }
-    let(:id) { nil }
-
-    it 'does not know what to do' do
-      expect(normalized_title).to eq ''
-    end
+    it { expect { normalized_title }.to raise_error Arclight::Exceptions::TitleNotFound }
```

Comment:
the coverage report says that the `message` isn't being tested here. you can add a second parameter that tests the message.